packages/contentstack-import/src/import/modules-js/labels.js

```diff
@@ -108,6 +108,15 @@ module.exports = class ImportLabels {
         })
         .catch(function (error) {
           // error while importing labels
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).label !== undefined) {
+            if (JSON.parse(error.request.data).label.name) {
+              message_content_type =
+                ' Failed label: ' + JSON.parse(error.request.data).label.name;
+            } 
+            error.errorMessage = error.errorMessage + message_content_type;
+          }
+          log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code